README.md

```diff
@@ -229,15 +238,76 @@ manually you can do so by opening a sql shell.
       --certs-dir=${HOME}/.cockroachdb/certs/clients/root \
       --host localhost
 
-#### 4a. Setup cms database:
+#### 4a. Setup PostgreSQL databasee
+
+Install PostgreSQL using the instructions found in the [PostgreSQL
+Documentation](https://www.postgresql.org/download/macosx/).
+
+Run the following commands to create the PostgreSQL certificates required for
+running PostgreSQL with Politeia.
```

Comment:
I'm not seeing it mentioned in the readme that it relies on openssl to generate the certs.

Reply:
Ops, I thought you meant to mention it in the `.sh` script, adding a note to the readme as well.